Carbon/Functions/Disable-NtfsCompression.ps1

```diff
@@ -30,12 +32,17 @@ function Disable-CNtfsCompression
     .EXAMPLE
     Disable-CNtfsCompression -Path C:\Projects\Carbon
 
-    Turns off NTFS compression on and decompresses the `C:\Projects\Carbon` directory, but not its sub-directories/files.  New files/directories will get compressed.
+    Turns off NTFS compression and decompresses the `C:\Projects\Carbon` directory (if compression is enabled), but not its sub-directories/files.  New files/directories will get compressed.

```

Comment:
Why do new files/directories still get compressed if compression on the directory is disabled?

Reply:
Copy and paste error from `Enable-CNtfsCompression` documentation.